phylogeny/prototypeSelection.py

```diff
@@ -511,7 +519,9 @@ def prototype_selection_constructive_maxdist(dm: DistanceMatrix,
     # the element to remove first is the one that has smallest distance to all
     # other. "Removing" works by tagging its distance-sum as infinity. Plus, we
     # decrease the number of available elements by one.
-    minElmIdx = currDists.argmin()
+    minElmIdx = next(x for x in currDists.argsort()
```

Comment:
I think I found a more elegant / fast way to implement this functionality. I also thought that the name 'seedset' might be more appropriate, thus I renamed the parameter accordingly. See my PR #66 
The idea is to artificially assign highest distance values to the elements of the seedset and than run the existing algorithm unchanged. Thus, it will have the same runtime for empty seedsets and only very minor additional (linear) runtime for seedsets.